src/components/TextInput/BaseTextInput/index.native.tsx

```diff
@@ -410,7 +410,7 @@ function BaseTextInput(
                                     }}
                                 />
                             )}
-                            {!!inputProps.isLoading && (
+                            {inputProps.isLoading !== undefined && (
```

Comment:
This change will be truthy when `inputProps.isLoading = false`. Do we want that?